changes/498.feature.rst

```diff
@@ -0,0 +1 @@
+Expose the icon and recipient data for :class:`Invite`\'s which target a GroupChannel.
```

Comment:
```suggestion
Expose the icon and recipient data for :class:`Invite`\'s which will target a GroupChannel.
```
I had missed the grammar here; I do agree with this https://github.com/DisnakeDev/disnake/pull/600/files#r914154831 comment however.

Reply:
This grammar is correct. We're talking about invite objects which currently target group channels, instead of ones that will at some point in the future.